src/components/Users.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react'
+
+import withAuthorization from './withAuthorization'
+import { db } from '../firebase'
+import UserList from './UserList'
+
+class Users extends Component {
+  state = { users: null }
+
+  componentDidMount() {
+    db.onceGetUsers().then(snapshot => this.setState({ users: snapshot.val() }))
+  }
+
+  render() {
+    const { users } = this.state
+
+    return (
+      <div>
+        <h1>Home</h1>
+        <p>The Home Page is accessible by every signed in user.</p>
+
+        {!!users && <UserList users={users} />}
```

Comment:
If users is an object (and according to `UserList` props, it is), then it will always be truthy.